src/ducks/balance/components/AccountRowLoading.jsx

```diff
@@ -8,7 +9,7 @@ import styles from './AccountRow.styl'
 import stylesLoading from './AccountRowLoading.styl'
 
 const AccountRowLoading = ({ t, institutionSlug }) => (
-  <li className={styles.AccountRow}>
+  <li className={cx(styles.AccountRow, stylesLoading.noLink)}>
```

Comment:
The base component should have the `u-clickable` class, and this one should remove it. This way we don't add a class for this purpose.